### PR TITLE
feat: split into files & add types for 23

### PR DESCRIPTION
**Pull request by weichweich**
update types for runtime 23.

@tjwelde for how long can we continue to but the types in each other? I'm not 100% happy with this practice. 😓 What is your opinion on this?

## Checklist:

- [ ] I have verified that the code works
- [ ] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,795 +1,15 @@
...
+import { types8 } from "./types_8";
+import { types9 } from "./types_9";
+import { types10 } from "./types_10";
+import { types12 } from "./types_12";
+import { types17 } from "./types_17";
+import { types18 } from "./types_18";
+import { types19 } from "./types_19";
+import { types20 } from "./types_20";
+import { types21 } from "./types_21";
+import { types23 } from "./types_23";
```
Comment (tjwelde) [inline src/index.ts:12]: these still need to be exported from here, so a third-party can use them individually (what we do in most applications)

**Review by tjwelde — CHANGES_REQUESTED** [review]
Yeah, don't love it. My bigger gripe is though, that it is hard to verify the types.
Also do we even need types for old runtimes, from before the launch? Maybe we should consolidate them.

File: src/index.ts (hunk 1 above)
Comment (weichweich) [reply src/index.ts:12]: fixed? 09fe5c3

File: src/index.ts [hunk 2]
```diff
@@ -1,795 +1,26 @@
...
+export { types8 } from "./types_8";
+export { types9 } from "./types_9";
+export { types10 } from "./types_10";
+export { types12 } from "./types_12";
+export { types17 } from "./types_17";
+export { types18 } from "./types_18";
+export { types19 } from "./types_19";
+export { types20 } from "./types_20";
+export { types21 } from "./types_21";
+export { types23 } from "./types_23";
```
Comment (tjwelde) [inline src/index.ts:23]: you already imported them, so you can directly export them again
Like
```
export { types20, types21, types23, ... }
```

**Review by tjwelde — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (tjwelde) [reply src/index.ts:12]: almost

File: src/index.ts (hunk 2 above)
Comment (weichweich) [reply src/index.ts]: 9d32f87

Comment (tjwelde) [reply src/index.ts]: yes, you are there.

**Review by tjwelde — APPROVED** [review]
LGTM

**Pull request merged**